src/Symfony/Bundle/FrameworkBundle/Kernel/MicroKernelTrait.php

```diff
@@ -142,6 +142,10 @@ public function registerContainerConfiguration(LoaderInterface $loader)
             }
 
             $container->setAlias(static::class, 'kernel')->setPublic(true);
+
+            if (!$container->hasParameter('container.dumper.inline_factories')) {
+                $container->setParameter('container.dumper.inline_factories', false);
```

Comment:
wouldn't it be better to default `container.dumper.inline_factories` to false when the parameter is not given? you changed this to true in https://github.com/symfony/symfony/pull/34872/files#diff-f7b23d463cba27ac5e4cb677f2be7623R154. so just reverting this would make the default behavior as wanted or?